app/views/content/salaries-and-benefits.md

```diff
@@ -56,57 +56,52 @@ keywords:
 
-## Primary and secondary teaching salaries
+## Primary and secondary teacher salary
 
-All qualified primary and secondary teachers will have a starting salary of at least £30,000. This will be higher for teachers working in London. 
+You’ll start on at least £30,000 (or more in London) as a primary or secondary school teacher in England. 
 
-### Qualified teacher salary
+Your salary will be reviewed every year, with most teachers moving up the pay range annually (this will depend on your school’s performance management arrangements).    
 
-Your school will have their own pay scales for qualified teachers. Pay increases will always be linked to performance, not length of service, and will be reviewed every year.
+A typical teacher with 5 years’ experience will earn at least £41,333 (or more in London).
 
-The teacher pay scales for qualified teachers are:
+### Qualified teacher salary
 
 | Area                                     | Minimum | Maximum |
 | -------                                  | -----   | -----   |
-| England (excluding London)     | £30,000 | £41,333 |
-| London fringe                            | £31,350 | £42,689 |
-| Outer London                             | £34,514 | £46,001 |
-| Inner London                             | £36,745 | £47,666 |
-
-Most established teachers will earn more than this maximum by progressing onto the upper pay range for teachers, or by becoming a leading practitioner.
+| England (excluding London)     | £30,000 | £46,525 |
+| London fringe                            | £31,350 | £47,839 |
+| Outer London                             | £34,514 | £51,179 |
+| Inner London                             | £36,745 | £56,959 |
 
-If you’re not sure whether you'll be teaching in inner, outer or the fringe of London, talk to your school.
+### Additional payments
 
-### Upper pay range for teachers
+You may also receive additional payments on top of your regular salary for taking on extra responsibilities. 
 
-If you can demonstrate excellence against all of the teacher standards, you can be put onto the upper pay range for teachers.
+This could be a one-off payment for a specific project, for example, creating a timetable. 
 
-This is for teachers who make a sustained and substantial contribution to their school.
+Or it might be an extra responsibility attached to your job role, for example, being the head of department. 
 
-The upper pay ranges for teachers are:
+Depending on your responsibilities, you could earn up to an extra £15,690. Your school will determine how much you receive. 
 
-| Area                                     | Minimum | Maximum |
-| -------                                  | -----   | -----   |
-| England (excluding London)     | £43,266 | £46,525 |
-| London fringe                            | £44,579 | £47,839 |
-| Outer London                             | £47,592 | £51,179 |
-| Inner London                             | £52,526 | £56,959 |
+## Career progression
```

Comment:
@gemmadallmandfe @philspray created a new section on career progression to try and further reiterate that the salaries already mentioned are what most teachers can expect to be on, and the salaries in this section (leading practitioner, headteachers) are for if you move further up the ranks (as the teacher pay team have said not many teachers are on the leading practitioner salary)